### ISSUE TITLE
Added non-passive event listener to a scroll-blocking 'touchstart' event

### ISSUE THREAD

**Issue by yooouuri** [robbinworks/v-siema#2]
I am getting the following warning:

> Added non-passive event listener to a scroll-blocking 'touchstart' event. Consider marking event handler as 'passive' to make the page more responsive. See https://www.chromestatus.com/feature/5745543795965952

**Comment by robbinworks** [comment]
I cannot seem to reproduce this warning, I've tested in the wild on latest `v-siema@0.0.6`:

OS: macOS Mojave (10.14.3)
Browser: Version 72.0.3626.121 (Official Build) (64-bit)

What environment are you experience this warning in?

Googling this error it seems the `touchstart` event handler api has been updated, some useful links:

- https://stackoverflow.com/a/46542547
- https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md

This has to be updated in [siema itself](https://github.com/pawelgrzybek/siema). It should probably be reported over at that repo's issue board as well in order to be noticed and fixed. As soon as this has been fixed in that repo I'll update this package to the latest version of siema.
